### PR TITLE
feat: initial support for ACR tasks

### PR DESCRIPTION
**Pull request by vlahane**
## Description

initial support for ACR tasks

## Updating an existing module

- [ ] This is a bug fix:
  - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
  - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
- [x] I have run `brm validate` locally to verify the module files.
- [x] I have run deployment tests locally to ensure the module is deployable.
- [ ] I have read the [Updating an existing module](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#updating-an-existing-module) section in the contributing guide and updated the `version.json` file properly:
  - [x] The PR contains backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`.
  - [ ] The PR contains backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] The PR contains breaking changes, and I have bumped the MAJOR version in `version.json`.
- [x] I have updated the examples in README with the latest module version number.

### CODE REVIEW

**Comment by vlahane** [comment]
@shenglol @dciborow  will you please review, thanks! 😄

File: modules/compute/container-registry/README.md [hunk 1]
```diff
@@ -58,7 +59,7 @@ This module deploys Container Registry (Microsoft.ContainerRegistry/registries)
 An example of how to deploy Azure Container Registry using the minimum required parameters.
 
 ```bicep
-module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
+module containerRegistry 'br/public:compute/container-registry:0.1.0' = {
```
Comment (dciborow) [inline modules/compute/container-registry/README.md:62]:
```suggestion
module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
```

File: modules/compute/container-registry/README.md [hunk 2]
```diff
@@ -75,7 +76,7 @@ An example of how to deploy a 'Premium' SKU instance of Azure Container Registry
 param subnetId string
 param privateDnsZoneId string
 
-module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
+module containerRegistry 'br/public:compute/container-registry:0.1.0' = {
```
Comment (dciborow) [inline modules/compute/container-registry/README.md:79]:
```suggestion
module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
```

File: modules/compute/container-registry/README.md [hunk 3]
```diff
@@ -97,7 +98,7 @@ module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
 An example of how to deploy a 'Premium' SKU instance of Azure Container Registry with replication to multiple Azure Locations.
 
 ```bicep
-module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
+module containerRegistry 'br/public:compute/container-registry:0.1.0' = {
```
Comment (dciborow) [inline modules/compute/container-registry/README.md:101]:
```suggestion
module containerRegistry 'br/public:compute/container-registry:1.0.1' = {
```

**Comment by shenglol** [comment]
Since we are adding a new parameter with a default value, we should bump the MINOR version and update the version for the examples in README.

File: modules/compute/container-registry/README.md [hunk 4]
```diff
@@ -41,15 +41,16 @@ This module deploys Container Registry (Microsoft.ContainerRegistry/registries)
...
-| name              | string | The name of the Azure Container Registry.                          |
-| loginServer       | string | The login server URL of the Azure Container Registry.              |
+| Name                | Type     | Description                                                        |
+| :------------------ | :------: | :----------------------------------------------------------------- |
+| `resourceGroupName` | `string` | The resource group the Azure Container Registry was deployed into. |
+| `resourceId`        | `string` | The resource ID of the Azure Container Registry.                   |
+| `name`              | `string` | The name of the Azure Container Registry.                          |
+| `loginServer`       | `string` | The login server URL of the Azure Container Registry.              |
 
 ## Examples
```
Comment (Gordonby) [inline modules/compute/container-registry/README.md:55]: Would be nice to have an example for ACR tasks in the readme. 👌

File: modules/compute/container-registry/README.md (hunk 3 above)
Comment (dciborow) [reply modules/compute/container-registry/README.md:129]:
```suggestion
module containerRegistry 'br/public:compute/container-registry:1.1.1' = {
```

File: modules/compute/container-registry/README.md (hunk 2 above)
Comment (dciborow) [reply modules/compute/container-registry/README.md:79]:
```suggestion
module containerRegistry 'br/public:compute/container-registry:1.1.1' = {
```

File: modules/compute/container-registry/README.md [hunk 5]
```diff
@@ -58,7 +59,7 @@ This module deploys Container Registry (Microsoft.ContainerRegistry/registries)
 An example of how to deploy Azure Container Registry using the minimum required parameters.
 
 ```bicep
-module containerRegistry 'br/public:compute/container-registry:1.0.3' = {
+module containerRegistry 'br/public:compute/container-registry:0.1.0' = {
```
Comment (dciborow) [inline modules/compute/container-registry/README.md:62]:
```suggestion
module containerRegistry 'br/public:compute/container-registry:1.1.1' = {
```

**Review by dciborow — APPROVED** [review]

**Pull request merged**